analysis/README.md

```diff
@@ -0,0 +1,38 @@
+# Distbench analysis script
+
+The distbench analysis tool is a python script developed to transform the results from the native distbench proto format to different formats that can be used for compabality with existing analysis tools.
+
+## How to use it
+
+To use this script, numpy should be installed, one can do this on the linux terminal with the following command:
+
+```bash
+sudo apt install python3-numpy
+```
+
+The script generates a directory tree with summaries across all the clients and services.
+
+to invoke the script use the follwing command line:
+
+```bash
+    ./results_conversion -- --output_directory=output_directory --input_file=name_of_the_file
+```
+
+The script also supports different formats for the output file, and options such as include or not the warmup rpcs and include or not a header at the beggining of every file.
+
+## Flags
+- **--output_directory**: path of the directory inside of which the directory tree will be created. (type=string)
+- **--input_file**: path of the protobuf file that the script will read. (type=string)
+- **--output_format**: Selects the format in which the summaries are gonna be. (type=string)
+- **--supress_header**: by adding this flag, the summary files won't have a header. (type=bool)
+- **--include_warmups**: by adding this flags, the summary files while include the warmup rpc's. (type=bool)
+
+```bash
+   ./results_conversion -- --output_directory=$PWD/golden_results/client_server_test --input_file=$PWD/golden_results/trace_context_test/clique_sync_burst_10x16000x1024x1024-grpc_polling_inline.pb --output_format=statistics --supress_header
```

Comment:
don't use the golden stuff hre it will confuse people. that is for our internal unit tests, not for general usage.